uint32 can_id
uint8 lenth
uint8[8] data